src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc

```diff
@@ -44,8 +44,10 @@ NaviStartStreamRequest::NaviStartStreamRequest(
     , EventObserver(application_manager.event_dispatcher())
     , retry_number_(0) {
   SDL_AUTO_TRACE();
+  const ApplicationManagerSettings& settings =
+      application_manager_.get_settings();
   std::pair<uint32_t, int32_t> stream_retry =
```

Comment:
@AByzhynar this variable has been declared by A.Kutsan so I dont know why he didn't include header, but ok, if you insist I'll include it.